multiqc/modules/rsem.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
```

Comment:
This file appears to be a duplicate of `multiqc/modules/rsem/rsem.py` on brief inspection - I guess it's been added by mistake?